src/cfe.c

```diff
@@ -491,6 +503,36 @@ else
   *surface_runoff_depth_m = 0.0;
   *infiltration_depth_m = 0.0;
   }
+
+
+ // impermeable fraction due to frozen soil
+ double factor = 1;
+ double smc_ref = 0.249; // taken from noahmp soil_params file for sandy loam
+ double frzk = 0.15; // taken from noahmp GENPARM.TBL
+ int cv_frz = 3;
+
+ if (ice_fraction_schaake > 1.0E-2) {
+   
+   double frz_fact = smcmax/smc_ref * (0.412 / 0.468);
+   double frzx = frzk * frz_fact;
+   double acrt = cv_frz * frzx / ice_fraction_schaake;

```

Comment:
These are getting to non-specific names; how important is it to maintain descriptive variable names?

Reply:
They were simply taken from NOAH_MP, and I don't think if they have any specific meaning here other simple local variables split for code readability. If you have any suggestions, would be happy to change them.